spec/02-integration/04-admin_api/04-plugins_routes_spec.lua

```diff
@@ -241,6 +241,19 @@ for _, strategy in helpers.each_strategy() do
             assert.is_table(json.fields)
           end
         end)
+        it("returns nested records as arrays", function()
+          for plugin, _ in pairs(helpers.test_conf.loaded_plugins) do
+            local res = assert(client:send {
+              method = "GET",
+              path = "/plugins/schema/request-transformer",
```

Comment:
The plugin name is hard-coded here. The surrounding loop is useless (and the below assertions about `default:[]` need to be conditionally triggered).

Reply:
welp, I thought I copied only the inside of the block of the previous test, not the for loop